updates/ryan3.sql

```diff
@@ -0,0 +1,48 @@
+
+Create Table selektion_beziehung(ID int Not NULL AUTO_INCREMENT, Bezeichnung varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
+PRIMARY KEY (`ID`) 
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
```

Comment:
The new table `selektion_beziehung` should also have a ```UNIQUE KEY `Bezeichnung` (`Bezeichnung`)```

Reply:
done